charts/github-actions-runners/values.yaml

```diff
@@ -0,0 +1,35 @@
+# -- Declare the name of the runner deployment
```

Comment:
In your helpers.tpl you are referencing nameOverride and fullnameOverride but they are not in here - that means they are also missing from your documentation.  IMO every value that you are using anywhere in this stack should be here in this file and documented otherwise you have "hidden" functionality.

Reply:
Changed accordingly - Vars added and Helpers file modified to follow similar pattern to other charts